app/src/interfaces/useObservationApi.interface.ts

```diff
@@ -37,3 +37,64 @@ export interface IGetObservationResponse {
   id: number;
   data: ICreateObservationRequest;
 }
+
+export interface IGetBlocksListResponse {
+  id: number;
+  block_id: number;
+  number_of_observations: number;
+  start_time: string;
+  end_time: string;
+}
+
+export interface IGetObservationsListResponse {
+  blocks: IGetBlocksListResponse[];
+}
+
+/**
+ * Create blockObservation post object.
+ *
+ * @export
+ * @interface ICreateBlockObservationPostRequest
+ */
+export interface ICreateBlockObservationPostRequest {
+  metaData: IBlockObservationMeta;
+  tableData: IBlockObservationTableData;
+}
+
+export interface IBlockObservationMeta {
```

Comment:
you dont need this
we can just use `IBlockObservationForm` which is defined in BlockObservationForm.tsx